### PR TITLE
#22 Improve the exam schedule page.

### LINKED ISSUE #22: [Impl] Improve the exam schedule page.
Let's strict the UI design.
- [ ] Add time period for the time column like `10.30 am - 11.30 am`
- [ ] Add page number buttons as in the UI design.

### PR DESCRIPTION
**Pull request by samadhi-k**

### CODE REVIEW

File: src/pages/schedule-page.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import { useState } from 'react';
 import { useTheme } from '@mui/material/styles';
 import Box from '@mui/material/Box';
 import Table from '@mui/material/Table';
-import { Typography } from '@mui/material';
+import { Typography} from '@mui/material';
```
Comment (Malith-19) [inline src/pages/schedule-page.js]: Add a space after the `Typography`

**Comment by samadhi-k** [comment]
image

**Comment by Malith-19** [comment]
There are some conflicts with the main too.
Please check it.

File: src/pages/schedule-page.js [hunk 2]
```diff
@@ -113,6 +116,7 @@ export default function () {
         setPage(newPage);
     };
 
+
```
Comment (chathuranga-jayanath-99) [inline src/pages/schedule-page.js]: we can remove this additional new line

**Review by chathuranga-jayanath-99 — APPROVED** [review]

**Review by Malith-19 — APPROVED** [review]

**Pull request merged**